pkg/forwarder/transaction.go

```diff
@@ -28,36 +28,24 @@ type HTTPTransaction struct {
 	Headers http.Header
 	// Payload is the content delivered to the backend.
 	Payload *[]byte
-	// ErrorCount is the number of times this HTTPTransaction failed to be processed.
-	ErrorCount int
```

Comment:
Should we keep `ErrorCount` as a debug tool ? It could be useful to identify one type of transaction causing an endpoint to be closed.